nvflare/app_common/launchers/subprocess_launcher.py

```diff
@@ -47,15 +58,24 @@ def launch_task(self, task_name: str, shareable: Shareable, fl_ctx: FLContext, a
             command = self._script
             env = os.environ.copy()
             command_seq = shlex.split(command)
+            log_file = os.path.join(self._app_dir, f"{task_name}_logfile")
```

Comment:
If not using `launch_once`, and multiple tasks of the same name are done in succession, then the log file will be overwritten each time. Should we append/have a different log file for each task, or is this ok for now?

Reply:
@SYangster good catch!
I remember this in the beginning but at some point I forgot.

I'll do the append mode.